tests/functional/disaster-recovery/sc_arbiter/test_add_capacity.py

```diff
@@ -107,6 +112,13 @@ def test_cluster_expansion(
         )
         logger.info("All the workloads pods are successfully up and running")
 
+        # setup vm and write some data to the VM instance
```

Comment:
this code block (til line 160) is quite generic (it repeats itself in https://github.com/red-hat-storage/ocs-ci/pull/9548/files#diff-46f753f4e560b98288b4f1da2258354d49a5ad5d15b1e3110e64e02829a06ed5R26) and could be used for other tests that include VM workloads. Suggest moving it to a VM related helper file.

Reply:
As discussed offline, created issue https://github.com/red-hat-storage/ocs-ci/issues/11186 to track. Will be addressing this in the next release (4.19)

Reply:
+1 to Elad